### PR TITLE
Some changes and 2020.2.1

### PR DESCRIPTION
**Pull request by windsent**

### CODE REVIEW

File: roles/twig/templates/application.yml.j2 [hunk 1]
```diff
@@ -12,6 +12,7 @@ time-suffix: {{ twig.time_suffix | default('yyyyMMdd') }}
 session:
   default-limit: {{ twig.session.default_limit | default('50') }}
   use-x-correlation-header: {{ twig.session.use_x_correlation_header | default('true') }}
+  show-retransmits: {{ twig.session.show_retransmits | default('true') }}
```
Comment (sip3leo) [inline roles/twig/templates/application.yml.j2:15]: `sip3-twig-ce` doesn't use `session.call.show-retransmits` and requires only `session.show-retransmits`. That's why we may remove `session.call.show-retransmits` from template.

File: inventories/sip3-backend.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+  vars:
+    ansible_host: 26.03.19.23
+    ansible_user: root
+    ansible_port: 22
+    ansible_become: true
+
+    platform: centos
+    edition: ce
+    version: 2020.1.1
```
Comment (sip3leo) [inline inventories/sip3-backend.yml:13]: `2020.2.1`?

File: inventories/sip3-captain.yml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+            pcap:
+              dev: eth1
+  vars:
+    ansible_user: root
+    ansible_port: 22
+    ansible_become: true
+
+    platform: centos
+    edition: ce
+    version: 2020.1.1
```
Comment (sip3leo) [inline inventories/sip3-captain.yml:19]: `2020.2.1`?

File: inventories/sip3-captain.yml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+      - rtcp
+
+    captain:
+      pcap:
+        dev: eth0
+        bpf_filter: udp
+      sender:
+        uri: udp://192.168.1.2:15060
+      management:
+        uri: udp://192.168.1.2:15090
```
Comment (sip3leo) [inline inventories/sip3-captain.yml:31]: We need to fix usage of this property [here](https://github.com/sip3io/sip3-ansible/blob/nginx-and-2020.2.1/roles/captain/templates/application.yml.j2#L94).

File: roles/salto/templates/application.yml.j2 [hunk 5]
```diff
@@ -186,7 +186,7 @@ sip:
     duration-timeout: {{ salto.sip.register.duration_timeout | default('900000') }}
     transaction-exclusions: {{ salto.sip.register.transaction_exclusions | default('[]') }}
   {% endif -%}
-{% if "rtp" in features or "rtcp" in features -%}
-rtp-r:
-  cumulative-metrics: {{ salto.rtp_r.cumulative_metrics | default('true') }}
-{% endif -%}
+  {% if "rtp" in features or "rtcp" in features -%}
```
Comment (sip3leo) [inline roles/salto/templates/application.yml.j2:189]: `rtp-r` is on top level and problem was unwanted indent. As far as i know it'll be enough to add new line after previous `endif`.

File: roles/salto/templates/logback.xml.j2 [hunk 6]
```diff
@@ -11,13 +11,6 @@
...
 
-    <logger name="io.sip3" level="INFO" additivity="false">
-        <appender-ref ref="DEFAULT"/>
-        {% if "webhook" in features -%}
-        <appender-ref ref="WEBHOOK"/>
-        {% endif -%}
-    </logger>
-
     {% if "webhook" in features -%}
     <appender name="WEBHOOK" class="io.sip3.logback.WebhookAppender">
```
Comment (sip3leo) [inline roles/salto/templates/logback.xml.j2:15]: Starting from version 1.0.2 `logback-webhook-appender` configuration for appender was changed:
```xml
    <appender name="WEBHOOK" class="io.sip3.logback.WebhookAppender">
        <url>https://hooks.slack.com/services/....</url>
        <json>
            {
                "text": ":fire: Salto :fire:",
                "attachments": [
                    {
                        "text": "{message}"
                    }
                ]
            }
        </json>
        <interval>60</interval>
        <encoder>
            <pattern>%d{dd-MM-yyyy HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
        </encoder>
    </appender>
```

Filter section was removed and added property `interval` in seconds to set interval between messages.

File: inventories/sip3-captain.yml (hunk 4 above)
Comment (sip3leo) [reply inventories/sip3-captain.yml:31]: And also change default `pcap.bpf-filter` value [here](https://github.com/sip3io/sip3-ansible/blob/nginx-and-2020.2.1/roles/captain/templates/application.yml.j2#L21) to `udp or tcp`.

**Review by sip3leo — CHANGES_REQUESTED** [review]

File: inventories/sip3-captain.yml (hunk 4 above)
Comment (windsent) [reply inventories/sip3-captain.yml:31]: `udp` is defined in Installation guide.
`udp or tcp` accepted

File: roles/salto/templates/logback.xml.j2 (hunk 6 above)
Comment (windsent) [reply roles/salto/templates/logback.xml.j2:15]: `<logback-webhook.version>1.0.1</logback-webhook.version>` in 2020.2.1 release...
Has this release `interval`?

File: roles/salto/templates/application.yml.j2 (hunk 5 above)
Comment (windsent) [reply roles/salto/templates/application.yml.j2:189]: I was forced to make new lines
rtp-r was moved on top level

File: roles/salto/templates/logback.xml.j2 (hunk 6 above)
Comment (sip3leo) [reply roles/salto/templates/logback.xml.j2:15]: Yeap, sorry. We change it since 1.0.1 version not 1.0.2

File: inventories/sip3-backend.yml [hunk 7]
```diff
@@ -0,0 +1,34 @@
+all:
```
Comment (agafox) [inline inventories/sip3-backend.yml:1]: I don't think we need this file in the project

File: inventories/sip3-captain.yml [hunk 8]
```diff
@@ -0,0 +1,34 @@
+all:
```
Comment (agafox) [inline inventories/sip3-captain.yml:1]: I don't think we need this file in the project

**Pull request merged**